sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -220,4 +220,12 @@ class SparkSqlParserSuite extends PlanTest {
 
     intercept("explain describe tables x", "Unsupported SQL statement")
   }
+
+  test("SPARK-18106 analyze table") {
```

Comment:
Ur, @srinathshankar .

I understand the reason why you put this there, so I looked at the StatisticsSuite.scala in both `hive` module and `sql` module.
But, we will not compare the value for this test case. If it's a parsing only grammar testcase, I prefer to put this in `core`.

How do you think about that?